homeassistant/components/zwave_js/api.py

```diff
@@ -1970,6 +1970,10 @@ async def post(self, request: web.Request, device_id: str) -> web.Response:
         if "file" not in data or not isinstance(data["file"], web_request.FileField):
             raise web_exceptions.HTTPBadRequest
 
+        target = None
+        if "target" in data:
+            target = int(cast(str, data["target"]))
```

Comment:
There's no way to specify the parameters of the post data?

Reply:
if there is I'm not aware of it

Reply:
I could try typing data and see if that resolves it

Reply:
Yeah, I guess pure aiohttp doesn't offer that.

Reply:
Yeah and I can't type it properly - they use `MultiDictProxy` under the hood so I can't use a TypedDict, and mypy will still say that data["target"] may be a `FileField` so I would still have to cast it.